Dockerfile

```diff
@@ -18,12 +18,11 @@ RUN pip3 install -r requirements.txt && pip install uwsgi
 
 ENV DJANGO_SETTINGS_MODULE=core.settings.label_studio
 ENV LABEL_STUDIO_BASE_DATA_DIR=/label-studio/data
-ENV LABEL_STUDIO_PORT="8080"
 
 COPY . /label-studio
 RUN python3.8 setup.py develop
 
-EXPOSE ${LABEL_STUDIO_PORT}
```

Comment:
A I right in thinking that the this environment variable is no longer used anywhere then? In which case should the docs [here](https://github.com/heartexlabs/label-studio/blob/master/docs/source/guide/start.md#L45) and [here](https://github.com/heartexlabs/label-studio/blob/master/docs/source/guide/start.md#L68) be updated to no longer use `LABEL_STUDIO_PORT` but to use `PORT` instead?

Reply:
[LABEL_STUDIO_PORT](https://github.com/heartexlabs/label-studio/blob/master/docs/source/guide/start.md#run-label-studio-on-localhost-with-a-different-port) is used to run the app outside the container, so it is a primary variable to explicitely set the port during the runtime. On the other hand, `PORT` variable is somewhat specific only to Heroku deployment (may be also some other cloud providers, but haven't checked it yet), so that's why a separate `Dockerfile.heroku` is created. As I mentioned before, LS app is looking for `LABEL_STUDIO_XXX` variables first to avoid catching the wrong global configurations.

Reply:
Are you sure about that? I can't find any reference to that env var anywhere in the project other than the start.md doc
![image](https://user-images.githubusercontent.com/13225609/113710513-b85c6800-96db-11eb-8082-84cf678f3590.png)

Reply:
Yes, [there is a logic](https://github.com/heartexlabs/label-studio/blob/master/label_studio/core/utils/params.py#L75) to resolve all env variables with prefixes

Reply:
Ah ok thanks for clarifying!

So am I understanding it correctly that now _either_ `LABEL_STUDIO_PORT` _or_ `PORT` will work? (In which case it should work on Heroku again)

Reply:
yes, but `LABEL_STUDIO_PORT` is taken first